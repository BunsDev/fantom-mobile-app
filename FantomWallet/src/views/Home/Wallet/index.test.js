import React from 'react';
import { shallow } from 'enzyme';

// eslint-disable-next-line import/first
import { WalletScreen } from './index';

test('renders correctly', () => {
  const wrapper = shallow(
    <WalletScreen balance="123.123" publicKey="0x00000123" isLoading={false} />
  );
  expect(wrapper).toMatchSnapshot();
});
