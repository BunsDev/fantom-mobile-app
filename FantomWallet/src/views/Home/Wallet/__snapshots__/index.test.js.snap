// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "rgb(14,14,18)",
      "flex": 1,
    }
  }
>
  <Image
    resizeMode="contain"
    source={
      Object {
        "testUri": "../../../src/images/BackgroundIcon.png",
      }
    }
    style={
      Object {
        "alignItems": "center",
        "height": "85%",
        "justifyContent": "center",
        "opacity": 0.03,
        "position": "absolute",
        "right": "-15%",
        "top": 0,
        "width": "45%",
      }
    }
  />
  <Connect(WalletNavigationBar)
    balance="123.123"
    isLoading={false}
    onRefresh={[Function]}
  />
  <View
    pointerEvents="box-none"
  />
</View>
`;
